packages/react/src/components/ImpactBadge/index.tsx

```diff
@@ -36,7 +26,6 @@ const ImpactBadge = forwardRef<HTMLDivElement, ImpactBadgeProps>(
         ref={ref}
         {...other}
       >
-        <Icon type={iconByType[type]} />
```

Comment:
With the removal of the Icon in the badge do we need to adjust the CSS to remove the extra padding? 
<img width="346" height="200" alt="image" src="https://github.com/user-attachments/assets/8c246c1f-d210-4cba-817c-48a10cfbb590" />

It's not quite centered in the badge container.

Reply:
That's coming from Badge: https://github.com/dequelabs/cauldron/blob/4d3cd93fc5161192e4a65afd007f19d29d0c2afd/packages/styles/badge.css#L25

Reply:
Should that centered? Or is that intentional?